python/test/test.py

```diff
@@ -223,6 +224,8 @@ def test_trades(exchange, symbol):
         time.sleep(delay)
         # dump(green(exchange.id), green(symbol), 'fetching trades...')
         trades = exchange.fetch_trades(symbol)
+        if trades:
+            testTrade(exchange, trades[0], symbol, int(time.time() * 1000))
```

Comment:
We need to run a loop here instead of checking just the first trade. That loop should be encapsulated in the `testTrades` function, which we should probably detach and transpile as well, like

- `test.trades.js` → `test_trades.py`
- `test.trades.js` → `test_trades.php`